### PR TITLE
Esbuild bundle the Python pool setup code

### PR DESCRIPTION
**Pull request by hoodmane**
We want to execute a single standalone JavaScript file in the pool scope. It should work in a normal v8 isolate with no modifications. This adds the esbuild bundle step, but keeps everything else working the same.

### CODE REVIEW

**Comment by danlapid** [comment]
Tagging @fhanau for Bazel changes

**Review by danlapid — APPROVED** [review]

File: WORKSPACE [hunk 1]
```diff
@@ -274,6 +274,13 @@ http_archive(
     url = "https://github.com/aspect-build/rules_ts/releases/download/v3.0.0/rules_ts-v3.0.0.tar.gz",
 )
 
+http_archive(
```
Comment (fhanau) [inline WORKSPACE:277]: - [ ] Since we recently added update-deps.py for workerd, we should be using that for all new dependencies

Comment (hoodmane) [reply WORKSPACE]: Great to see a reduction in the number of differences between developing with workerd vs internally. How's it look now?

Comment (fhanau) [reply WORKSPACE]: Thank you, LGTM! Didn't look closely at js_file but this should be good to go based on Dan's review

Comment (hoodmane) [reply WORKSPACE]: Well Dan wrote `js_file.bzl` =) I should probably add him as a coauthor on the commit.

**Pull request merged**